waldo/app/controllers/ChatController.js

```diff
@@ -34,7 +34,7 @@ class ChatController {
                     res.json({
                         data: {
                             id: conversation._id,
-                            type: "Coversation"
```

Comment:
Conversation was spelled wrong, and I think it makes more sense to call it type chat since this is the chat endpoint and ember likes a type of "chat" and it doesn't really affect anything on the backend